index.html

```diff
@@ -2181,6 +2181,159 @@ <h4>Presentations Including Holder Claims</h4>
+This specification defines the following <a>property</a> for the expression of a
+data schema, which can be included by an <a>issuer</a> in
+the <a>verifiable credentials</a> that it issues:
+        </p>
+
+        <dl>
+          <dt><var>credentialSchema</var></dt>
+          <dd>
+            <p>
+The value of the <code>credentialSchema</code> <a>property</a> MUST be one or
+more data schemas that provide <a>verifiers</a> with enough information to
+determine if the provided data conforms to the provided schema(s). Each
+<code>credentialSchema</code> MUST specify its <code>type</code> (for example,
+<code>JsonSchema2023</code>), and an <code>id</code> <a>property</a>
+that MUST be a <a>URL</a> identifying the schema file. The precise contents of
+each data schema is determined by the specific type definition.
+            </p>
+            <p>
+If multiple schemas are present, validity is determined according to the
+processing rules outlined by each associated <code>credentialSchema</code>
+<code>type</code> property.
+            </p>
+          </dd>
+        </dl>
+
+        <p class="note">
+The <code>credentialSchema</code> <a>property</a> provides an opportunity to
+annotate type definitions or lock them to specific versions of the vocabulary.
+Authors of <a>verifiable credentials</a> can include a static version of their
+vocabulary using <code>credentialSchema</code> that is locked to some content
+integrity protection mechanism. The <code>credentialSchema</code>
+<a>property</a> also makes it possible to perform syntactic checking on the
+<a>credential</a> and to use <a>verification</a> mechanisms such as JSON Schema
+[[?VC-JSON-SCHEMA-2023]] validation.
+        </p>
+
+        <pre class="example nohighlight"
+          title="Usage of the credentialSchema property to perform JSON schema validation">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "id": "http://university.example/credentials/3732",
+  "type": ["VerifiableCredential", "ExampleDegreeCredential", "ExamplePersonCredential"],
+  "issuer": "https://university.example/issuers/14",
+  "validFrom": "2010-01-01T19:23:24Z",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "degree": {
+      "type": "ExampleBachelorDegree",
+      "name": "Bachelor of Science and Arts"
+    },
+    "alumniOf": {
+      "name": "Example University"
+    }
+  },
+  <span class="highlight">"credentialSchema": [{
+    "id": "https://example.org/examples/degree.json",
+    "type": "JsonSchema2023"
+  },
+  {
+    "id": "https://example.org/examples/alumni.json",
+    "type": "JsonSchema2023"
+  }]</span>
+}
+        </pre>
+
+        <p>
+In the example above, the <a>issuer</a> is specifying a
+<code>credentialSchema</code>, which points to a [[?VC-JSON-SCHEMA-2023]] file that
+can be used by a <a>verifier</a> to determine if the
+<a>verifiable credential</a> is well formed.
```

Comment:
```suggestion
<a>verifiable credential</a> is valid.
```

Reply:
I cringe at what "well formed" might be interpreted as... the intention is for the issuer and the verifier to agree that the data matches a schema... this is data validation... "well forming" would apply more broadly, for example to the design of the claims, if maps or arrays were used, casing considerations, etc...